### PR TITLE
Invoice updates

### LINKED ISSUE #3152: Enterprise invoice CSS updates
See: https://github.com/ParabolInc/action/pull/3158 for current implementation

AC
No Figma design required, we think we can jump straight to implementation

- [ ] Current invoice implementation updated to match Parabol brand standards

effort: 12 points

EDITS: @jordanh adjusted AC

### LINKED ISSUE #3153: Design Discount line in invoices
AC
- [ ] Our invoices support a line for discounts, if any are present. it's probably a line between subtotal & total & probably in red
- [ ] Implementation made directly on branch, or, design made in Figma and implementation issue created

Effort: 12 points

### PR DESCRIPTION
**Pull request by ackernaut**
### Test
- [ ] Adds tier badge to invoice, fixes #3152 
- [ ] Adds styled discount coupon information, if any, fixes #3153 
- [ ] Cleans up deprecated usage of tags and colorPalette
- [ ] Scopes reflection card prompt footer to discuss phase for delightful animations in grouping

### CODE REVIEW

**Comment by ackernaut** [comment]
@mattkrick I updated our tags and added the Pro or Enterprise tag to the invoice.

As for the discount coupons, I’m unsure if we want to handle that as line items with other line items, or its own section above the total. Feel free to jump on this branch to add that, and I could make a style pass once we have a pattern.

**Comment by ackernaut** [comment]
@mattkrick okay this is ready for your eyes, especially the handling of the new `invoice.discount.coupon` data. Not sure what we need, but for UI I used `name`, `amount_off` and `percent_off`. You can see the coupon object in the [Stripe API](https://stripe.com/docs/api/discounts) of course. I also got rid of the old tag by colorPalette pattern for named tags.

File: packages/server/graphql/types/Coupon.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The name of the discount coupon from Stripe'
+    },
+    percent_off: {
+      type: GraphQLInt,
+      description: 'The percent off the invoice, if any'
+    },
+    valid: {
```
Comment (mattkrick) [inline packages/server/graphql/types/Coupon.ts]: why do we need this?

File: packages/server/graphql/types/Coupon.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const Coupon = new GraphQLObjectType({
+  name: 'Coupon',
+  description: 'The discount coupon from Stripe, if any',
+  fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The ID of the discount coupon from Stripe'
+    },
+    amount_off: {
```
Comment (mattkrick) [inline packages/server/graphql/types/Coupon.ts]: nit: camel case over snake case for data that we control

**Comment by ackernaut** [comment]
@mattkrick this is ready for another pass of review

**Comment by ackernaut** [comment]
@mattkrick any more suggestions beyond the PR?

**Comment by mattkrick** [comment]
good to go!
punting on migrating old invoices with coupon codes that don't show until it becomes a problem. filed #3230 as a separate non-blocking issue

**Pull request merged**